modules/commons/src/main/resources/contexts/neurosciencegraph/core/data/v0.3.0.json

```diff
@@ -537,6 +537,12 @@
       "morphology": {
```

Comment:
I suggest to bump the version of this context to v0.3.1

Reply:
done